packages/auth/e2e/helpers.js

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable no-console */
+const { getE2eTestProject, getE2eEmulatorHost } = require('../../app/e2e/helpers');
+const http = require('http');
+
+exports.getRandomPhoneNumber = function getRandomPhoneNumber() {
+  return '+593' + Utils.randString(9, '#19');
+};
+
+exports.clearAllUsers = async function clearAllUsers() {
```

Comment:
Off topic but curious if a disable user API could be reverse engineered of the Emulator Web UI requests as well, does the emulator web ui have an option to disable a user?

Reply:
Oh it does! This might solve the problem. I will check if it's possible

Reply:
Looks like a POST to the standard APIs works against the emulator! The docs were not clear on that, they have an "emulator specific" setting but don't mention if the non-specific APIs work, apparently they do:

```
scheme | http
-- | --
host | localhost:9099
filename | /identitytoolkit.googleapis.com/v1/accounts:update

{"disableUser":true,"localId":"tUkcA81QgKn0l6n1CXl3EFW1kR1x"}
```

Needs the same 'Authorization: Bearer owner' trick from account purge REST API call, but I already have the infrastructure to do REST API calls to the emulator so this is easy peasy now.

Reply:
Nice!